bodhi/tests/server/services/test_updates.py

```diff
@@ -932,6 +932,83 @@ def test_provenpackager_request_update_with_none_ci(self, publish, *args):
             res.json_body['errors'][0]['description'],
             config.get('not_yet_tested_msg'))
 
+    @mock.patch(**mock_taskotron_results)
+    @mock.patch(**mock_valid_requirements)
+    @mock.patch('bodhi.server.notifications.publish')
+    def test_provenpackager_request_update_with_gw_satisfied(self, publish, *args):
+        """Ensure provenpackagers can request changes for any update which
+        satisfies Greenwave policies"""
```

Comment:
This one actually asserts that the reason the user can't push is not Greenwave. I forgot to update the comment above line 967, :-)